app/assets/javascripts/application/subscription.coffee

```diff
@@ -0,0 +1,11 @@
+$('#subscribe').submit ->
```

Comment:
in rails when we use `remote: true` it will automatically do all those stuff
``` js
 $.ajax(
   url: $(this).attr('action')
   type: 'POST'
  data: input)
```
so we just need to catch success and error responses in this case:
`$("#subscribe").on "ajax:success", (e, response) ->` or
 `$("#subscribe").on "ajax:error", (e, response) ->`

Also it would be good to call this script only after document is ready and only on page which contains this element - but because it was first js file we have in our app let me refactor it after I merge it so i can show you and the rest of the team how I would like us to write js (a good practice kind of thing ;))